.github/workflows/sync.yml

```diff
@@ -78,7 +78,7 @@ jobs:
 
       - name: Upload sync log file artifact
         if: ${{ always() }}
-        uses: actions/upload-artifact@v4
+        uses: actions/upload-artifact@v4-${{ matrix.pipeline }}}
```

Comment:
Should this be in the file name, not the action name?

Reply:
oops, copy-paste mistake...